tests/test_questions.py

```diff
@@ -441,30 +441,39 @@ def test_get_error_messages_unknown_key(self):
 
     def test_get_error_messages(self):
         question = self.question().filter(self.context())
-        ordered_dict = question.get_error_messages({'example': [
-            {'field': 'yesno', 'index': 0, 'error': 'answer_required'},
-            {'field': 'evidence', 'index': 0, 'error': 'answer_required'},
-            {'index': 1, 'error': 'answer_required'}
-        ]})
-
-        errors = list(ordered_dict.items())
-        assert errors == [
-            ('yesno-0', {
-                'input_name': 'yesno-0',
-                'message': 'There was a problem with the answer to this question',
-                'question': u'First Need-yesno'
-            }),
-            ('evidence-0', {
-                'input_name': 'evidence-0',
-                'message': 'There was a problem with the answer to this question',
-                'question': u'First Need-evidence'
-            }),
-            ('example', {
-                'input_name': 'example',
-                'message': 'There was a problem with the answer to this question',
-                'question': 'Dynamic list'
+
+        errors_and_messages = {
```

Comment:
None afaik: this was the existing test, which I had to fix because I'd changed the message - so it made sense to then test both messages here, if I was changing it anyway. Don't know why that one was chosen.